Project1- Using gspread library to populate a google sheet

LogicAL007 opened the pull request.

LogicAL007 committed:
submission

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,26 @@
+populate_worksheet("Ecommerce store")
+worksheet_list = spreadsheet.worksheets()
+print(worksheet_list)
+# worksheet= spreadsheet.worksheet("Ecommerce store")
```
remove redundant comments

elseagle commented on omotosho-project-c92e4f4bc071.json:
```diff
@@ -0,0 +1,13 @@
+{
```
don't commit a credential file. Use .env with `decouple` or `python-dotenv` library. Ensure you add your .env to .gitignore

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,26 @@
+spreadsheet = gc.open("Gspread practice")
+# sh = gc.create('A new spreadsheet')
+
+def get_or_create_worksheet(spreadsheet, Ecom , rows=1, cols=1):
```
add docstring to explain what your code does

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,26 @@
+        worksheet = spreadsheet.add_worksheet(Ecom, rows, cols)
+    return worksheet
+
+def populate_worksheet(Ecom):
```
some docstring or comment will help here

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,26 @@
+
+populate_worksheet("Ecommerce store")
+worksheet_list = spreadsheet.worksheets()
+print(worksheet_list)
```
don't use print, checkout the `logging` library and replace all print statements with it

elseagle reviewed (changes requested):
Good start @LogicAL007 - added some comments.

Kindly use descriptive titles for your PRs and commits. Also, could you add description so we can have a summary of what you've built? 

A readme markdown file to give detail insight to your project

LogicAL007 committed:
Corrections

 I made corrections to the issues addressed ;
-don't commit a credential file
-add docstring to explain what your code does
-A readme markdown file to give detail insight to your project

LogicAL007 committed:
Delete omotosho-project-c92e4f4bc071.json

LogicAL007 committed:
include logging

this commit removes the print as instructed and generates log instead

LogicAL007 committed:
Merge branch 'Omotosho_Ayomide' of https://github.com/Data-Epic/Omotosho_Ayomide into Omotosho_Ayomide

LogicAL007 committed:
Merge branch 'main' into Omotosho_Ayomide

LogicAL007 committed:
Merge branch 'main' into Omotosho_Ayomide

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,51 @@
+
+logging.basicConfig(level=logging.INFO)
+
+def get_or_create_worksheet(spreadsheet, Ecom , rows=1, cols=1):
```
don't use title format for parameters "Ecom" should be "ecom"

elseagle commented on my_gspread.py:
```diff
@@ -0,0 +1,51 @@
+    
+    :param Ecom: The title of the worksheet to populate.
+    '''
+    worksheet = get_or_create_worksheet(spreadsheet, Ecom, rows=df.shape[0]+1, cols=df.shape[1])
```
can you comment on this line to help us know why you are doing `df.shape[0]+1` ? It is unclear to everyone else except , help us make it clear to anyone that picks your code

LogicAL007 committed:
A comment was added to explain the df.shape[0]+1 and the parameter were updated

- change title format for parameters "Ecom" should be "ecom"

-comment on this line to help us know why you are doing df.shape[0]+1.

LogicAL007 committed:
Merge branch 'Omotosho_Ayomide' of https://github.com/Data-Epic/Omotosho_Ayomide into Omotosho_Ayomide

tobaadesugba commented on README.md:
```diff
@@ -1,6 +0,0 @@
-# Populating an excel sheet with Gspread library
```
Your readme is a little incomplete. Include details like the author of the project, the tools you used, the source of the data used, and other details you feel are important.

LogicAL007 committed:
A documentation was added to explain what the script does

LogicAL007 committed:
resolve merging issue
